easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.12.0-fosscuda-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,108 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow'
+version = '1.12.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+builddependencies = [
+    # must stick to Bazel 0.18, cfr. https://github.com/tensorflow/tensorflow/issues/23380
+    ('Bazel', '0.18.0'),
+    ('protobuf', '3.6.1'),
+]
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('wheel', '0.31.1', versionsuffix),
+    ('h5py', '2.8.0', versionsuffix),
+    ('cuDNN', '7.1.4.18'),
+    ('NCCL', '2.3.7'),
+    ('TensorRT', '4.0.1.6', versionsuffix),
+]
+
+use_pip = True
+
+exts_list = [
+    ('protobuf-python', '3.6.1', {
+        'modulename': 'google.protobuf',
```

Comment:
`modulename` should move down, ideally in line with existing easyconfigs, cfr. https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.12.0-foss-2018b-Python-3.6.6.eb#L28